iteration2_test_ex24.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from iteration2_exercise24 import(
+    strings_contain_only_letters,
+    equal_string_length,
+    strings_contain_same_letters,
+)
+
+class AnagramChecker(unittest.TestCase):
+    """Tests to see if two strings are anagrams"""
+
+    def test_only_letters(self):
+        """Test to see if both strings contain only letters"""
+
+        user_string_1 = 'note'
+        user_string_2 = 'tone'
+
+        strings_with_only_letters = strings_contain_only_letters(user_string_1, user_string_2)
```

Comment:
Be careful with your variable names. This name tells me that this variable contains multiple strings that only have letters, perhaps in a list. But that's not the case at all. It is saying whether or not, true or false, if both strings contain only letters. A better name is `strings_only_contain_letters`. That tells the reader everything they need to know. I think a good exercise for you is going through this entire PR and ensuring your variable names are communicating what they do to the reader of the code.